libs/langchain-community/src/vectorstores/neo4j_vector.ts

```diff
@@ -4,10 +4,20 @@ import type { EmbeddingsInterface } from "@langchain/core/embeddings";
 import { VectorStore } from "@langchain/core/vectorstores";
 import { Document } from "@langchain/core/documents";
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Any = any;
+
 export type SearchType = "vector" | "hybrid";
 
+export enum IndexType {
```

Comment:
We generally try to avoid `enum` due to bundler/environment issues.

Reply:
Refactored to plain type `export type IndexType = "NODE" | "RELATIONSHIP";`